cristin-project/src/main/java/no/unit/nva/cristin/projects/FetchOneCristinProject.java

```diff
@@ -30,7 +33,7 @@ public FetchOneCristinProject() {
 
     @JacocoGenerated
     public FetchOneCristinProject(Environment environment) {
-        this(new CristinApiClient(), environment);
+        this(new CristinApiClient(CristinAuthenticator.getHttpClient()), environment);
```

Comment:
Why are you using this on an open endpoint?

Reply:
Same as above